src/components/Subscriptions/SubscriptionsMonthlyGood.vue

```diff
@@ -248,17 +274,40 @@ export default {
 		result({ data }) {
 			this.isMonthlyGoodSubscriber = data?.my?.autoDeposit?.isSubscriber ?? false;
 			if (this.isMonthlyGoodSubscriber) {
-				const autoDepositAmount = parseFloat(data?.my?.autoDeposit?.amount ?? 0);
-				this.donation = parseFloat(data?.my?.autoDeposit?.donateAmount ?? 0);
+				const activeMonthlyGoodSubscription = data?.mySubscriptions?.values
+					.find(sub => sub?.category?.subscriptionType === 'MG');
+				this.donation = parseFloat(activeMonthlyGoodSubscription?.donation ?? 0);
 				this.dayOfMonth = data?.my?.autoDeposit?.dayOfMonth;
 				this.category = data?.my?.monthlyGoodCategory ?? '';
-				this.mgAmount = autoDepositAmount - this.donation;
+				this.mgAmount = parseFloat(activeMonthlyGoodSubscription?.amount ?? 0);
 				this.paymentMethod = data?.my?.autoDeposit?.paymentMethod ?? {};
 				this.subscriptionId = data?.my?.autoDeposit?.id;
+				this.firstName = data?.my?.userAccount?.firstName ?? '';
+				this.subscriptionsLoans = data?.mySubscriptions?.values?.length ?? 0;
```

Comment:
This is not correct. There will only be 1 value in `data?.mySubscriptions?.values` of type monthly good. I am not sure that we currently have a way to determine number of loans made. 

I think you'll have to take the subscription start date, the subscription amount and todays date, and manually count how many times it would have triggered for each $25 loan purchase to find out the number of loans being lent. This assumes that MG purchases only $25 loan share on different loans....